src/Tribe/Service_Providers/Processes.php

```diff
@@ -10,28 +10,42 @@
 class Tribe__Service_Providers__Processes extends tad_DI52_ServiceProvider {
 
 	/**
-	 * Binds and sets up implementations.
+	 * An array of all handler actions registered in this HTTP request.
+	 *
+	 * This is an array cache with a request lifetime by design.
+	 *
+	 * @var array
+	 */
+	protected $handler_actions;
+
+	/**
+	 * An array of all queue actions registered in this HTTP request.
+	 *
+	 * This is an array cache with a request lifetime by design.
+	 *
+	 * @var array
+	 */
+	protected $queue_actions;
+
+	/**
+	 * Hooks the filters and binds the implementations needed to handle processes.
 	 */
 	public function register() {
-		if ( ! (
-			tribe( 'context' )->doing_ajax()
-			&& false !== $action = tribe_get_request_var( 'action', false )
-		) ) {
+		// If the context of this request is neither AJAX or Cron bail.
+		if ( ! ( tribe( 'context' )->doing_ajax() || tribe( 'context' )->doing_cron() ) ) {
```

Comment:
I'd personally rather see a `$context` setup from `tribe( 'context' )` and use that for these two method calls. Not required but would be nice.

Reply:
Good idea, thanks.